config/default.yml

```diff
@@ -14,6 +14,12 @@ toolkit:
   build:
     dist:
       root: 'dist'
+      keep:
```

Comment:
Please create a folder documentation and include a readme file to document the build process with the new configuration.

Reply:
`./vendor/bin/run toolkit:build-dist --help`